spec/features/search/search_by_movie_title_spec.rb

```diff
@@ -0,0 +1,31 @@
+# As an authenticated user,
+# When I visit the '/discover' path
+# I should see
+#
+#  Button to Discover top 40 movies
+# Details When the user clicks on the top 40 button they should be taken to the movies page.
+#
+#  A text field to enter keyword(s) to search by movie title
+#  A Button to Search by Movie Title
+# Details When the user clicks on the Search button they should be taken to the movies page
+#
+# The movies will be retrieved by consuming The MovieDB API
+
+require "rails_helper"
+
+RSpec.describe "As an authenticated user, When I visit the '/discover' path I should see" do
```

Comment:
Can we update this describe to be more like "Discover Landing Page"
Then have another describe block under it with "As an authenticated user"

Reply:
Yes!  Most definitely!